src/events/ingress.py

```diff
@@ -72,10 +79,24 @@ def _on_ingress_revoked(self, _: IngressPerAppRevokedEvent):
     def _on_auth_proxy_removed(self, _: AuthProxyRelationRemovedEvent):
         """Handle the removal of the AuthProxy."""
         self.logger.info("AuthProxy configuration gone")
-        self.history_server.update(self.context.s3, self.context.ingress)
+        self.history_server.update(
+            self.context.s3, self.context.ingress, self.context.authorized_users
```

Comment:
```suggestion
            self.context.s3, self.context.ingress, None
```

shouldn't this be none?